CExcercises/Ex3-genericAdder.c

```diff
@@ -15,9 +14,10 @@ El resultado será la suma de todos los field1 de cada estructura S del arreglo
 #include <stdio.h>
 #include <string.h>
 
-#define offsetOf(STRUCT, FIELD) ((int)(&STRUCT) - (int)(&FIELD))
+#define offsetOf(STRUCT, FIELD) ((int)(&FIELD) - (int)(&STRUCT))
 
 int genericPlus(const void* array, size_t elementSize, size_t fieldOffset, size_t amountOfElements);
```

Comment:
Do you mean the macro, like:
    #define static ...
That is not possible because the compiler just replace the value by define, does not make any static control.